pyfakefs/fake_filesystem_unittest.py

```diff
@@ -461,7 +461,7 @@ def is_fs_function(fct):
                 return False
 
         module_names = list(self._fake_module_classes.keys()) + [PATH_MODULE]
-        for name, module in set(sys.modules.items()):
+        for name, module in list(sys.modules.items()):
```

Comment:
I don't remeber why I used a set here - there shouldn't be any duplicates, so I guess this will not change the behavior.